tests/wandb_run_test.py

```diff
@@ -758,3 +773,45 @@ def test_init_with_settings(live_mock_server, test_settings):
         == wandb_sdk.wandb_settings.Source.INIT
     )
     run.finish()
+
+
+def test_repo_job_creation(live_mock_server, test_settings):
+    with open("test.py", "w") as f:
```

Comment:
same as the other comment

Reply:
This one can't be removed, because log_code does nothing, I can add filesystem isolation